pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -294,24 +295,49 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     }
   }
 
-  protected async syncDockerCompose() {
-    await Promise.all([
-      this.syncHostDockerCompose(),
-      ...(await this.supportedDistros).map(distro => this.syncDistroDockerCompose(distro.name)),
-    ]);
+  protected async syncDockerPlugins() {
+    const wslPluginNames = await this.getWslDockerCliPluginNames();
+    const hostPluginNames = await this.getHostDockerCliPluginNames();
+
+    const promises: Promise<void>[] = wslPluginNames.map(wslPluginName => this.syncHostDockerPlugin(wslPluginName) );
```

Comment:
Is this going backwards? `wsl` plugins are being passed to `syncHostDockerPlugin()`, and `host` plugin names are being passed to `syncDistroDockerPlugin()`.

Reply:
Whoops, good catch